src/main/java/net/mcreator/minecraft/ElementUtil.java

```diff
@@ -208,7 +207,7 @@ public static List<MItemBlock> loadAllPointOfInterest(Workspace workspace) {
 				.forEach(modElement -> elements.add(
 						((VillagerProfession) modElement.getGeneratableElement()).pointOfInterest));
 		DataListLoader.loadDataList("villagerprofessions").stream().filter(e -> e.isSupportedInWorkspace(workspace))
-				.filter(typeMatches("poi")).map(e -> (MCItem) e).toList()
+				.filter(typeMatches("other")).map(e -> (MCItem) e).toList()
 				.forEach(v -> elements.add(new MItemBlock(workspace, v.getName())));
```

Comment:
This is not the way you check for the `other` parameter of a DataListEntry.
```suggestion
		DataListLoader.loadDataList("villagerprofessions").stream().filter(e -> e.isSupportedInWorkspace(workspace))
				.filter(DataListEntry::getOther).forEach(poi -> elements.add(new MItemBlock(workspace, (String) poi)));
```